unlock-app/src/components/interface/locks/Manage/elements/Members.tsx

```diff
@@ -120,12 +123,31 @@ export const Members = ({
     )
   }
 
+  const checkoutLink = `/locks/checkout-url?lock=${lockAddress}&network=${network}`
+
   if (noItems && (hasSearch || hasActiveFilter)) {
```

Comment:
I think we should only show the buttons to airdrop and/or generate purchase URL if there is no active filter (i.e the fact that there are no results is because there are no keys, not because none was "found")